src/assets/data/MockData.js

```diff
@@ -0,0 +1,57 @@
+import { v4 as uuidv4 } from 'uuid'; 
```

Comment:
use static ID, please
It might trigger unnecessary UI updates while you don't have real API

I won't recommend using this package at all
Leave it for the node.js
there are very specific situations where you would want to use it on frontend (like adding a new Movie, though in real life it'll still happen on backend)

Reply:
Yes I only use it when creating a new Movie via the form. This was the fastest solution to work with mock data while we aren;t working with the api yet